orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/JarDiffsTask.groovy

```diff
@@ -121,6 +122,7 @@ class JarDiffsTask implements DiffTask {
     RestAdapter restAdapter = new RestAdapter.Builder()
       .setEndpoint(address)
       .setClient(new OkClient(okHttpClient))
+        .setErrorHandler(SpinnakerRetrofitErrorHandler.getInstance())
```

Comment:
I'm surprised spotless doesn't complain about this.
```suggestion
      .setErrorHandler(SpinnakerRetrofitErrorHandler.getInstance())
```

Reply:
Code formatted as per the suggestion.